aether-producer/producer/aether_producer.py

```diff
@@ -196,10 +196,10 @@ def connect_aether(self):
                 if not self.kernel:
                     self.logger.info('Connecting to Aether...')
                     with KernelHandler(self):
-                        self.kernel = KernelClient(
-                            url=self.settings['kernel_url'],
-                            username=self.settings['kernel_admin_username'],
-                            password=self.settings['kernel_admin_password'],
+                        self.kernel = Client(
+                            self.settings['kernel_url'],
+                            self.settings['kernel_admin_username'],
+                            self.settings['kernel_admin_password'],
```

Comment:
Do you know what I would suggest here too? :nerd_face:

Reply:
Ah! I updated it in docker-compose but not in the usage. :cry: Now the tests will fail...

Reply:
_life is cruel_

Reply:
Nevermind! Still will do.

Reply:
I'll need to remember to tell @MrPink that we're changing the required environment variable as this will break production.

Reply:
Oops!